src/ScottLogic.Internal.Training/ScottLogic.Internal.Training.Matcher/Order.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace ScottLogic.Internal.Training.Matcher
+{
+    public class Order : IEquatable<Order>
+    {
+        public int AccountNumber { get; }
+        public int Quantity { get; set; }
+        public int Price { get; }
+        public string Action { get; }
+        public int TimeRank {get;}
+
+        public Order(int accountNumber, int quantity, int price, string action, int timeRank)
```

Comment:
If action is only every going to be "buy" or "sell" I would make an enum and use that to make it more explicit.

E.g,

public enum OrderTypes{
    None,
    Buy,
    Sell
}

depending on the circumstance I would generally put an invalid value as the first (default) value. This means that we can check to see if it can be set or not. If we didn't include the None type here, any value created but not set explicitly  would be set to Buy, which could cause problems.

Reply:
so now in the matcher I should check
if(currentOrder.action == OrderType.Sell)
...
else if (currentOrder.action == OrderType.Buy)
...
else { error, no action set}
?